src/fairchem/core/_cli_hydra.py

```diff
@@ -32,29 +33,48 @@
 
 
 class Submitit(Checkpointable):
```

Comment:
Just out of curiosity, since the `Runner` class is already a `Checkpointable`, do we need both the `Runner` and `Submitit` to inherit from `Checkpointable`?

Reply:
Runner is not a Checkpointable?